src/main/flight/pid.c

```diff
@@ -171,6 +172,8 @@ static FAST_RAM filterApplyFnPtr dtermNotchFilterApplyFn;
 static FAST_RAM void *dtermFilterNotch[2];
 static FAST_RAM filterApplyFnPtr dtermLpfApplyFn;
 static FAST_RAM void *dtermFilterLpf[2];
+static FAST_RAM filterApplyFnPtr dtermLpf2ApplyFn;
+static FAST_RAM void *dtermLpf2Filter[2];
```

Comment:
Please make the naming uniform with existing code, e.gm `dtermFilterLpf2` instead.
Or better, let's rename old and new to `dtermLowpass` and `dtermLowpass2`. Why have the world filter twice in one name? :)

Reply:
`lowpass`/`lowpass2` is also the naming I went with in the dual-stage PR.

Reply:
makes me wonder if we should be using `FAST_RAM` in gyroSensor for similar storage as well...

Reply:
Don't dive into that, I'm working on it as a part of F7 optimization branch

Reply:
OK will change to to dtermLowpass and dtermLowpass2